src/desktop/desktop.js

```diff
@@ -37,6 +37,10 @@ export default class Desktop extends Component
     {
         this.setState({ isFull: true });
     }
+    stopFullScreen()
```

Comment:
Could you tell me what does this function do? If you feel fullScreen() function written by me is also not used then probably remove it. As I think the fullscreen is being handled within child IIRC

Reply:
The fullscreen code you wrote was not working, the state change was directly accepted and used by the react fullscreen package. I have cleaned up the code by removing those functions